tests/test_examples/test_example_metadata.py

```diff
@@ -8,18 +9,18 @@
 
 
 @pytest.mark.parametrize("file, example", _metadata_examples().items())
-def test_schema_example_filenames(file, example):
+def test_schema_example_filenames(file: str, example: dict) -> None:
     """Assert that all examples are .yml, not .yaml"""
     assert file.endswith(".yml")
 
 
 @pytest.mark.parametrize("file, example", _metadata_examples().items())
-def test_validate(file, example):
+def test_validate(file: str, example: dict) -> None:
```

Comment:
```suggestion
@pytest.mark.parametrize("example", _metadata_examples().values())
def test_validate(example: dict[str, Any]) -> None:
```

Other way around here, with `values()`. The parametrization requires that the string parameter names (`"example"`) match the function argument name. Haven't testing these, but they should be OK if `items()` worked.